CONTRIBUTING.md

```diff
@@ -1,60 +1,69 @@
-# Contributing guide for Glasskube
+# Contributing guide for Glasskube 🧊
 
 Welcome, and thank you for deciding to invest some of your time in contributing to the Glasskube project!
 The goal of this document is to define some guidelines to streamline our contribution workflow.
 
-## Getting started
+## Before you get started ✋
+---
+There are many types of issues you can take on when contributing to the Glasskube project. We try our best to provide a wide array of open issues that vary in levels of complexity. From beginners to seasoned developers, everyone should be able to find something to work on.
 
-### Where should I start?
+### Let's find the perfect open issue for you!
 
 - If you are new to the project, please check out the [good first issue](https://github.com/glasskube/glasskube/labels/good%20first%20issue) label.
-- If you are looking for something to work on, check out our [open issues](hhttps://github.com/glasskube/glasskube/issues).
+- If you are ready to make a big impact on the project, check out the [current milestone](https://github.com/glasskube/glasskube/milestones) that is being worked on and filter the issues by `"help-wanted"`, these issues are the ones that will make it into the next official release. 
+- If you are looking for something specific to work on, check out our [open issues](hhttps://github.com/glasskube/glasskube/issues) and filter against the available tags such as `component: cli`, `component: ui` `component: repo`, `documentation`.
 - If you have an idea for a new feature, please open an issue, and we can discuss it.
 - We are also happy to help you find something to work on. Just reach out to us.
```

Comment:
It might make sense to explicitly exclude issues that are already assigned to someone or have the https://github.com/glasskube/glasskube/issues?q=is%3Aissue+is%3Aopen+label%3Aneeds-triage label